### PR TITLE
Carrier Billing APIs Readiness Checklist MetaRelase Fall24

### LINKED ISSUE #163: Update API Readiness Checklist format and information for Meta-Release Fall24
**Problem description**
API Readiness Checklist format has been update by Release Management WG: https://github.com/camaraproject/ReleaseManagement/blob/main/documentation/API-Readiness-Checklist.md

**Expected action**
It is needed to adapt this checklist for the APIs involved in MetaRelease Fall24:
- Carrier Billing Payment
- Carrier Billing Refund

Document as a reference: 
https://github.com/camaraproject/CarrierBillingCheckOut/blob/main/documentation/API_documentation/Carrier%20Billing-API-Readiness-Checklist.md

**Additional context**
It is proposed to keep the same file and document within it two tables, one per API.

### PR DESCRIPTION
**Pull request by PedroDiez**
#### What type of PR is this?

Add one of the following kinds:
* documentation
* subproject management

#### What this PR does / why we need it:

PR to update API readiness Checklist for Carrier Billing Family APIs
PR shows expected output for MetaRelease 

#### Which issue(s) this PR fixes:

Fixes #163 

#### Special notes for reviewers:

To be merged after #165 is MERGED and Issues #159, #162 are CLOSED

#### Changelog input

```
 Carrier Billing Family APIs Readiness Checklist for MetaRelease Fall24

```

#### Additional documentation 

N/A

### CODE REVIEW

**Comment by bigludo7** [comment]
Thanks @PedroDiez - will wait for the required missing pieces.

File: documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md
Comment (hdamker) [inline documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md:3]:
```suggestion
Checklist for Carrier Billing Refund v0.1.0 in r1.1
```

File: documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md
Comment (hdamker) [inline documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md:7]: Just as a hint, you can use relative links for these links, which are independent of the actual release tag, example:

```suggestion
|  1 | API definition                               |   M   |         M         |    M    |    M   |  Y   | [link](/code/API_definitions/carrier_billing_refund.yaml) |
```

**Comment by hdamker** [comment]
Looks good to me, see two hints above.

File: documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md
Comment (PedroDiez) [reply documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md:7]: Thanks for this comment indication

**Comment by PedroDiez** [comment]
@hdamker 
Intention with this PR was to have ready API readiness Checklist with M4 horizon. Given the fact that:
- User Stories have to be updated
- Initial Test Plan have to be provided

And be ready for M4 milestone.

So in this way M3 (Release Candidate PR): https://github.com/camaraproject/CarrierBillingCheckOut/pull/167, is focused on YAML.

Therefore, should I adapt this PR to "show" the status for M3 and generate afterwards a new PR for the M4 milestone, once User Stories and Tetsing Plan is provided?

**Comment by hdamker** [comment]
> Therefore, should I adapt this PR to "show" the status for M3 and generate afterwards a new PR for the M4 milestone, once User Stories and Testing Plan is provided?

Yes, that is the intention from Release Management perspective ... the checklist should reflect the current status of the (pre)release. And it's ok if the User Stories and Test Definitions are "tbd" in the release candidate (the latter is an exception decided by the TSC for this release cycle). 

For the following releases, e.g. the one for M4, you only need to update the few details in the checklists as part of the Release PR, no separate PR needed for that.

File: documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md
Comment (hdamker) [inline documentation/API_documentation/Carrier Billing Refund-API-Readiness-Checkkist.md:3]: The actual API Version number could also been set within the release PR, but as you asked for review here:

```suggestion
Checklist for Carrier Billing Refund v0.1.0-rc.1 in r1.1
```

File: documentation/API_documentation/Carrier Billing-API-Readiness-Checklist.md
Comment (hdamker) [inline documentation/API_documentation/Carrier Billing-API-Readiness-Checklist.md:3]:
```suggestion
Checklist for Carrier Billing v0.3.0-rc.1 in r1.1.
```

(It's relevant as then the "release-candidate" column shows what should be in ... even if there is currently no difference to an initial public version)

**Review by hdamker — APPROVED** [review]
LGTM ... just two comments which can also be done in the release PR.

Link to changelog will be /CHANGELOG.md, permanently :-)

**Comment by PedroDiez** [comment]
I will do in this PR as I think it needs at least RM approval to be able to be merged

**Review by hdamker — APPROVED** [review]
LGTM from Release Management perspective

**Comment by hdamker** [comment]
> I will do in this PR as I think it needs at least RM approval to be able to be merged

In RM we are reviewing the checklist(s) together with the release PR.
As the checklists are now perfect, should I merge them for you? (otherwise you would one Codeowner approval).

**Comment by PedroDiez** [comment]
> > I will do in this PR as I think it needs at least RM approval to be able to be merged
> 
> In RM we are reviewing the checklist(s) together with the release PR. As the checklists are now perfect, should I merge them for you? (otherwise you would one Codeowner approval).

If possible, yes please, it is appreaciated if you can merge it Herbert

**Pull request merged**
